save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/AvatarController.kt

```diff
@@ -77,16 +122,14 @@ internal class AvatarController(
                 log.info("Saved $contentLength bytes of $avatarKey")
             }
         }
-        .flatMap {
-            blockingToMono {
```

Comment:
In this particular case boundedElastic scheduler does nothing for performance  and no difference with the default parallel scheduler  : even if we execute a database query in a separate thread with a thread pool, we won’t get much performance boost, because it is the last calculation in this operations chain (next map just returns). Anyway you will need to wait until the finish of db transaction.

Reply:
But you are right that in general it’s better to use bounded, so probably yes, need to return it back

Reply:
fixed